.github/actions/execute-assert-arc-e2e/action.yaml

```diff
@@ -0,0 +1,151 @@
+name: 'Execute and Assert ARC E2E Test Action'
+description: 'Queue E2E test workflow and assert workflow run result to be succeed'
+
+inputs:
+  github-auth-token:
```

Comment:
Why does this action expect a token while the other expects an app id and a pk? Can we standardise the input?

Reply:
the setup action takes the appId/pk and outputs a auth token, which gets used for the rest of the workflow.
Do you want me to make this action also take appid/pk and generate another token it can use?

Reply:
I think having the workflow determine which authentication method to use is better. Our actions can just use a token, without necessarily know whether it's a GITHUB_TOKEN, a PAT or an app installation token. WDYT?